build.sbt

```diff
@@ -15,7 +15,7 @@ lazy val versions = new {
 
 lazy val baseSettings = Seq(
   version := "1.0.0-SNAPSHOT",
-  scalaVersion := "2.11.8",
+  scalaVersion := "2.12.1",
```

Comment:
Instead of only building for 2.12, let's cross-build for both 2.11 and 2.12. As a reference, we do that in Finatra here: https://github.com/twitter/finatra/blob/develop/build.sbt